src/runtime-rs/crates/runtimes/src/manager.rs

```diff
@@ -149,6 +162,36 @@ impl RuntimeHandlerManagerInner {
 
         let config = load_config(spec, options).context("load config")?;
 
+        // Retrieve the log-levels set in configuration file, modify the FILTER_RULE accordingly
+        let mut agent_level = String::new();
+        for v in config.agent.values() {
```

Comment:
Seems it could be better to use:
```
if let Some(hypervisor_config) = config.hypervior.get(config.runtime.hypervisor_name) {
...
}
```

We are now allowing multi hypervisor configs written in one configuration file.
Same to agent.

Also suggest wrap the whole modification (from L165-L194) into one function.